metadata.rb

```diff
@@ -65,18 +65,6 @@
           :dscription => "mysql user account for hive",
           :type => "string"
 
-attribute "hive2/metastore/private_ips",
```

Comment:
If you were to upgrade a cluster with Karamel, and you didn't want to re-run the Hive recipe, but needed to know the private_IP of the hive services, you would need these, right?

Reply:
We shouldn't be using the metastore ip anywhere anymore. it should be using consul.